api/Pipfile

```diff
@@ -7,6 +7,7 @@ verify_ssl = true
 ipython = "*"
 pycodestyle = "*"
 pytest-django = ">=3.5"
+pytest-raises = "*"
```

Comment:
Is this new dependency being used at all?

Reply:
It is used here: https://github.com/WordPress/openverse-api/pull/859/files#diff-3f72d9dc992edb46dcea0b7136c9ecaa0f888653a2c4c6261ba0116de468eef8R16